docs/HyperIndex/Guides/configuration-file.mdx

```diff
@@ -77,20 +70,10 @@ More information on Human Readable ABI parsing is available [here](https://docs.
 - Address field should contain the address of the proxy contract, which emits the events on the specified blockchain.
 - Should the human readable ABI format not be used, then the ABI which is referenced in config file needs to be copied from the implementation contract into the specified abi directory.
 
-## Example `config.yaml` from Greeter template using ReScript language:
 
-```yaml
-name: Greeter
-description: Greeter indexer
-networks:
-  - id: 137 # Polygon
-    start_block: 45336336
-    contracts:
-      - name: PolygonGreeter
-        abi_file_path: abis/greeter-abi.json
-        address: 0x9D02A17dE4E68545d3a58D3a20BbBE0399E05c9c
-        handler: ./src/EventHandlers.bs.js
-        events:
-          - event: NewGreeting
-          - event: ClearGreeting
-```
+import CodeBlock from '@theme/CodeBlock';
+import Schema from "@site/static/schemas/config.evm.json";
+import JSONSchemaViewer from "@theme/JSONSchemaViewer";
+
+## Config Schema Explorer
+<JSONSchemaViewer schema={ Schema } />
```

Comment:
Is it possible to add Fuel schema and make it closed by default?

Reply:
Do you think it should be in the same doc? Will add it for sure 😄

Reply:
Hey @DZakh, this plugin is not that configurable. I can hack it but think it's not worth the squeeze, it's meant to be a quick implementation.

No out of the box way to configure closed by default.

Do you think I still just add it on this doc open?

Reply:
Or maybe just add it to the fuel page?

Reply:
Let's skip it for now